client/src/screens/patient-portal/Appointments/Calendar/index.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+// eslint-disable-next-line import/order
+import FullCalendar from "@fullcalendar/react"; // this import should be at the top
+import dayGridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction"; // needed for dayClick
+import timeGridPlugin from "@fullcalendar/timegrid";
+import PropTypes from "prop-types";
+
+function renderEventContent(eventInfo) {
+  return (
+    <>
```

Comment:
`<>` is it necessary here?

Reply:
No, it isn't necessary. but It can come in handy in the future if we would like to render more than just a <p>.